tests/memex/storage_test.py

```diff
@@ -407,10 +434,10 @@ def fetch_annotation(patch):
 
 
 @pytest.fixture
-def models(patch):
-    models = patch('memex.storage.models', autospec=False)
-    models.Annotation.return_value.is_reply = False
-    return models
+def patched_annotation(patch):
+    klass = patch('memex.storage.models.Annotation')
+    klass.return_value.is_reply = False
+    return klass
```

Comment:
The issue was that we patched the whole `models` module, so even the `update_document_metadata.assert_called_once_with` was mocked out and always truthy. I think the only fix is to not patch the whole `models` module. Unless you have a better way to make sure that these tests can fail (by commenting out lines in the implementation)?